gammapy/obs/observation.py

```diff
@@ -85,3 +91,244 @@ def select_linspace_subset(self, num):
         # Round down to nearest integer
         indices = indices.astype('int')
         return self[indices]
+
+    def select_obs_var_range(self, selection_variable,
+                             value_min, value_max, inverted=False):
+        """Make an observation table, applying some selection.
+
+        Generic function to apply a 1D box selection (min, max) to a
+        table on any variable that is in the observation table and can
+        be casted into a `~astropy.units.Quantity` object.
+
+        If the inverted flag is activated, the selection is applied to
+        keep all elements outside the selected range.
+
+        Parameters
+        ----------
+        selection_variable : string
+            name of variable to apply a cut (it should exist on the table)
```

Comment:
I always start these one-line descriptions with a capital letter, I think most others in Numpy / Scipy / Astropy do this as well. Please fix all cases you can quickly find (OK to make unrelated code changes in this case ... it's cleanup for the upcoming release).